src/routes/layout.scss

```diff
@@ -0,0 +1,22 @@
+@use "uswds-core" as * with (
+  $theme-show-notifications: false
+);
+
+/* For some reason the border between the header and the rest of the page is 
+    placed on the next sibling of the header, but only if it's a .usa-section
+    or <main>, presuambly since those are full-width elements. Since there's no
+    comments or guidance around this we'll override this rule and apply it to
+    the bottom of the header itself, which seems like a more sane approach.
+   USWDS Source: 
+    https://github.com/uswds/uswds/blob/c397204e35a20cc31e9a2da9567cac8d0214264a/packages/usa-header/src/styles/_usa-header.scss#L58-L64
+*/
```

Comment:
Yeah, I suppose that makes sense. But then again there's the `usa-header--extended` class which probably should have handled this. It is what it is; we've got the border working with this override and it's documented in case things break in the future.